packages/node/src/node/server/InteractionServer.ts

```diff
@@ -781,16 +757,18 @@ export class InteractionServer implements ProtocolHandler, InteractionRecipient
 
     async handleInvokeRequest(
         exchange: MessageExchange,
-        { invokeRequests, timedRequest, suppressResponse, interactionModelRevision }: InvokeRequest,
+        request: InvokeRequest,
         messenger: InteractionServerMessenger,
         message: Message,
     ): Promise<void> {
+        const { invokeRequests, timedRequest, suppressResponse, interactionModelRevision } = request;
         logger.debug(
-            `Received invoke request from ${exchange.channel.name}${invokeRequests.length > 0 ? ` with ${invokeRequests.length} commands` : ""}: ${invokeRequests
-                .map(({ commandPath: { endpointId, clusterId, commandId } }) =>
-                    this.#context.structure.resolveCommandName({ endpointId, clusterId, commandId }),
-                )
-                .join(", ")}, suppressResponse=${suppressResponse}`,
+            () =>
+                `Received invoke request from ${exchange.channel.name}${invokeRequests.length > 0 ? ` with ${invokeRequests.length} commands` : ""}: ${invokeRequests
```

Comment:
I think I'd put invoke at NOTICE, this is seems like an event you'd want to audit

Reply:
as discussed above: we have the concrete invokes as info ... i think thats more important then to know why they came in ... so I would leave as debbug